doc/devel/contributing.rst

```diff
@@ -354,9 +354,7 @@ The definition of the pylab text function is a simple pass-through to
 
   # in pylab.py
   def text(*args, **kwargs):
-      ret = gca().text(*args, **kwargs)
-      draw_if_interactive()
-      return ret
+      return gca().text(*args, **kwargs)
```

Comment:
This was outdated. There is no explicit `draw_if_interactive()` involved anymore.